docs/proposals/accepted/observatorium.md

```diff
@@ -0,0 +1,222 @@
+  * Single Cluster (everything runs inside the cluster).
+  * Mix of in cluster monitoring/alerting and subset of/all metrics pushed to a central location.
+  * The purely centralized push model
+* Horizontally Scalable: supporting smaller as well as larger scale.
+  * It should be possible to deploy the solution in small environments such as single-node clusters.
+  * Scaling up is a matter of adding more instances rather than throwing more resources at the same instance.
+* Relatively pp, easy to operate, and ubiquitous storage even for long retention:
+  * Object storage is the main storage solution for longer retention.
+  * Block storage is only for short term data or local buffers.
+* Ability to define data retention policies.
+* Control over the granularity of the data access.
+  * Allowing a tenant to read a subset of data.
+  * Allowing a tenant to send a subset of the data off the cluster.
+* Secure by default
+  * All external-facing APIs support TLS for encryption
+  * All external-facing APIs support OpenID for authentication.
+  * All external-facing APIs support [OPA](https://www.openpolicyagent.org/) for authorization.
+* High availability and solid failover handling.
+  * For example, for the write path, the client side can survive the unavailability of a central cluster for e.g 30m without data loss.
+* Built-in mechanisms to support production setups out of the box.
+  * Rate-limiting.
+  * Retries.
+  * Sane defaults for metrics ingestion (remote write configuration, sharding, queues).
+* Ability to maintain alert routing (e.g Alertmanager).
+  * Alerts API:
+    * Alertmanager v2 API
+  
+## Soft Goals
+
+Currently not in scope due to time constraints, but possible if you want to contribute:
+
+* The ability for users to manage dashboards (e.g Grafana)
+* Provide a single access point with access to all signals
+  *  Discovery
+* Support per tenant API quota and usage reports (e.g. show-back, billing).
+* Support cold and hot storage.
+* Ability to ingest (write), store, query (read), and visualize traces.
+* Ability to ingest (write), store, query (read), and visualize profiles.
+* Standard out of box alerts and monitoring rules
+  * For the components that Observatorium will deploy
+  * For Kubernetes
+* Support Blackbox Probes.
+* Support [OLAP](https://en.wikipedia.org/wiki/Online_analytical_processing) Analytics Queries.
+* Ability to correlate signals and quickly navigate from one to another.
+* Ability to ingest metrics via pure push method (push gateway, OLTP metrics protocol).
+* Deploy the components on non-Kubernetes platforms.
+* Pluggability: Supporting more than one project per signal
+
+## Non-Goals
+
+Currently not in scope due to project complexity, but we may revisit:
+
+* Extensibility in Authn/Authz protocols.
+* Define packaging and installation methods (this will be covered by a separate design doc)
+* Non Observability Events.
+* Other APIs then defined above.
+
+## How
+
+TBD
```

Comment:
Anything to add here? Feels like an important point ^^

Reply:
nah ;p

Reply:
that's trivial ;p